app/decorators/visit_decorator.rb

```diff
@@ -1,6 +1,7 @@
 class VisitDecorator < Draper::Decorator
   delegate_all
   decorates_association :rejection
+  NO_VISITORS_IN_NOMIS = 'No record of this visitor in NOMIS'.freeze
```

Comment:
This text should not be hardcoded in english, we've talked to translate staff one day.